lumigator/backend/backend/services/workflows.py

```diff
@@ -217,11 +239,11 @@ def create_workflow(self, request: WorkflowCreateRequest) -> WorkflowResponse:
 
         return workflow
 
-    def delete_workflow(self, workflow_id: str) -> WorkflowResponse:
+    def delete_workflow(self, workflow_id: str, force: bool) -> WorkflowResponse:
```

Comment:
Needed so that I can delete a hanging job https://github.com/mozilla-ai/lumigator/issues/1093